peepingtom/datablocks/multiblocks/orientedpointblock.py

```diff
@@ -1,14 +1,25 @@
-import numpy as np
-
 from .multiblock import MultiBlock
 from ..simpleblocks import PointBlock, OrientationBlock
 
 
 class OrientedPointBlock(MultiBlock):
-    def __init__(self, positions: np.ndarray, orientations: np.ndarray, **kwargs):
+    def __init__(self, positions=(), orientations=(), pixel_size=None, **kwargs):
         super().__init__(**kwargs)
-        self.positions = PointBlock(positions)
+        self.positions = PointBlock(positions, pixel_size=pixel_size)
         self.orientations = OrientationBlock(orientations)
 
-    def __shape_repr__(self):
-        return f'{self.positions.data.shape}'
+    @property
+    def n(self):
+        return self.positions.n
```

Comment:
so n is the name we will use for the 'row' dimension across the codebase? just checking so we're on the same page

Reply:
Yes! I'm not set on it, if you have better ideas. I think it's convenient to have a quick-to-type, easily remembered thing that will work across the codebase, more or less